definitions/npm/keycode_v2.2.0/test_keycode_v2.2.0.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import keycode from 'keycode';
+
+let stringResult: string;
+let numberResult: number;
+// parses a code
+stringResult = keycode(1);
+// parses a key name
+numberResult = keycode('enter');
+// parses a keyboard event
+stringResult = keycode(new KeyboardEvent('keyboard'));
+
+// invalid input
+// $ExpectError
+keycode({});
+
+// can access isEventKey
+const boolResult: boolean = keycode.isEventKey(
+  new KeyboardEvent('keyboard'),
+  1
+);
+
+// can access raw objects
+keycode.codes;
+keycode.aliases;
+keycode.names;
```

Comment:
Could you please wrap tests in a few [`describe/it` blocks](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope)?